CRM/Core/Form.php

```diff
@@ -366,6 +366,9 @@ public function &add(
     $type, $name, $label = '',
     $attributes = '', $required = FALSE, $extra = NULL
   ) {
+    if ($type === 'radio') {
+      CRM_Core_Error::deprecatedFunctionWarning('CRM_Core_Form::addRadio');
```

Comment:
I added this in because at the moment CRM_Core_Form::add 's function declaration makes no sense for radios better to use the addRadio function instead